### ISSUE TITLE
[Request] Fog of War

### ISSUE THREAD

**Issue by decoytrix** [MidoriKami/Mappy#53]
Apologies if tis option was in the plugin and I missed it while looking. 
I was running Mount Rokkon and realized it was harder to determine unknown paths because the game revealed unknown/unseen areas.  I would like the option to see the fog of war, aka the dark cloud that covers areas you've never seen like in the normal game map.

**Comment by MidoriKami** [comment]
Definitely not going to add Fog of War to Mappy at any point.

There are a couple reasons for this,
First, Mappy is intended to be used by players that are already somewhat familiar with the game. I do not believe people new to the game should be using plugins like this.

Secondly, Fog of War is implemented as a shader which I frankly don't even know how to start tackling even if I did want this feature. (tldr; it's a very difficult thing to do even if I did want it).

**Comment by decoytrix** [comment]
New to the game? It was about the plugin working for completionists to track pathing on new, level 90 content (like Mount Rokken, the variant dungeon) or future expansion maps. 

But fair enough on the difficulty aspects. 
Would it be possible to set the plugin to work on a separate mappable keybind rather than fully replace the game's map?  That way, people can swap between the game's base map system and the plugin map depending on which was more useful to the situation and features needed without having to go through the xlplugins UI to disable Mappy.

**Comment by MidoriKami** [comment]
> New to the game? It was about the plugin working for completionists to track pathing on new, level 90 content (like Mount Rokken, the variant dungeon) or future expansion maps.

Systems like that would apply to all maps, what makes you think it would only effect high level content? 
Maps don't care about levels.

> Would it be possible to set the plugin to work on a separate mappable keybind rather than fully replace the game's map? 

No, if Mappy's features don't meet your needs, use the vanilla map.

**Comment by decoytrix** [comment]
>Systems like that would apply to all maps, what makes you think it would only effect high level content?
Maps don't care about levels. 

That was MY point, that the feature is of benefit to players "already familiar with the game," not just "people new to the game."  

>if Mappy's features don't meet your needs, use the vanilla map.

Don't get defensive. It's just sad when plugins loose features the base game provides rather than improving upon what is there, so I thought I'd suggest an alternative way of bypassing the limitations/downgrades.

**Comment by MidoriKami** [comment]
Comments removed.

Tldr; This is not a feature I want in my plugin.
